linkerd/core/src/main/scala/io/buoyant/linkerd/SvcConfig.scala

```diff
@@ -33,9 +33,11 @@ trait SvcConfig {
    * responseClassifier categorizes responses to determine whether
    * they are failures and if they are retryable.
    */
-  @JsonProperty("responseClassifier")
   var _responseClassifier: Option[ResponseClassifierConfig] = None
 
+  @JsonProperty("responseClassifier")
+  def responseClassifier_=(r: ResponseClassifierConfig): Unit = _responseClassifier = Some(r)
```

Comment:
yeah – I was going to add one, but the conventional name for it, `responseClassifier()`, clashes with the `responseClassifier()` def & I didn't want to rename that? I suppose it could be called `getResponseClassifier` but that doesn't seem v. idiomatic?